cpplint.py

```diff
@@ -1318,8 +1318,10 @@ def RepositoryName(self):
       # searching up from the current path.
       root_dir = current_dir = os.path.dirname(fullname)
       while current_dir != os.path.dirname(current_dir):
-        if (os.path.exists(os.path.join(current_dir, ".git")) or
-            os.path.exists(os.path.join(current_dir, ".hg")) or
+        if (os.path.exists(os.path.join(current_dir, ".git"))):
```

Comment:
pylint does not like the outer parentheses in this case, hence the travis failure.